examples/getting_started/2_Pipeline.ipynb

```diff
@@ -315,14 +315,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "tf.Images(tf.shade(agg_d3_d5.where(aggc.sel(cat='d3') == aggc.sel(cat='d5')), name=\"d3+d5 where d3==d5\"),\n",
-    "          tf.shade(      agg.where(aggc.sel(cat='d3') == aggc.sel(cat='d5')), name=\"d1+d2+d3+d4+d5 where d3==d5\"))"
+    "sel1 = agg_d3_d5.where(aggc.sel(cat='d3') == aggc.sel(cat='d5')).astype('uint32')\n",
+    "sel2 =       agg.where(aggc.sel(cat='d3') == aggc.sel(cat='d5')).astype('uint32')\n",
+    "\n",
+    "tf.Images(tf.shade(sel1, name=\"d3+d5 where d3==d5\"),\n",
+    "          tf.shade(sel2, name=\"d1+d2+d3+d4+d5 where d3==d5\"))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Here we're specifying `uint64` because at least current versions of xarray's `where` method convert to floating point, and keeping the type unsigned is important for `shade` to know that zero is a missing-data value here, to be rendered with a transparent background.\n",
```

Comment:
Noticed that `uint64` should be `uint32` here to match with the code cell.

Reply:
Good catch!

```suggestion
    "Here we're specifying `uint32` because at least current versions of xarray's `where` method convert to floating point, and keeping the type unsigned is important for `shade` to know that zero is a missing-data value here, to be rendered with a transparent background.\n",
```